optimum_benchmark/backends/pytorch/backend.py

```diff
@@ -224,6 +224,10 @@ def load_diffusers_model(self):
                 self.pretrained_model.vae.decode, **self.config.torch_compile_config
             )
 
+        # See https://github.com/huggingface/diffusers/issues/4649
+        if "xl" in self.config.model:
+            self.pretrained_model.unet.config.addition_embed_type = None
```

Comment:
what's the actual problem here ? reading the conversation it seems that users have this issue when they use the wrong pipeline for their target models (stable diffusion for an xl model) which is not something we wanna hack here.